wormhole_chain/proto/wormhole/tx.proto

```diff
@@ -30,4 +37,42 @@ message MsgRegisterAccountAsGuardian {
 message MsgRegisterAccountAsGuardianResponse {
 }
 
+// Same as from x/wasmd but with vaa auth
+message MsgStoreCode {
+  // Signer is the that actor that signed the messages
+  string signer = 1;
+  // WASMByteCode can be raw or gzip compressed
+  bytes wasm_byte_code = 2 [ (gogoproto.customname) = "WASMByteCode" ];
+  reserved 3, 4, 5;
```

Comment:
we are still using these fields, the vaa just contains the hash of `wasm_byte_code`.  Signer is needed for all the tx's

Reply:
I was referring to the reserved field ids (3, 4, and 5).  Usually reserved fields are used to maintain backwards compatibility when a field is removed (https://developers.google.com/protocol-buffers/docs/proto3#reserved).  Since this is a brand-new proto we don't really need to worry about backwards compatibility yet.

I think we don't strictly _need_ to reserve these to maintain compatibility with the x/wasm protos but it doesn't really cost us anything so I'm fine with leaving them in.  Maybe add a comment explaining why these are here.

Reply:
Yeah I was thinking to mirror x/wasm just in case but you're right, I don't see this ever being an issue.  I'll delete